tests/foreman/cli/test_satellitesync.py

```diff
@@ -81,6 +81,23 @@ def export_import_cleanup_module(target_sat, module_org):
     )
 
 
+@pytest.fixture(scope='class')
+def docker_repo(module_target_sat, module_org):
+    product = make_product({'organization-id': module_org.id})
+    repo = make_repository(
+        {
+            'organization-id': module_org.id,
```

Comment:
Can we use DockerRepository from repository mixins here to create and sync docker repo?